src/Sdk/RSWebApi/Contracts/AcquireJobRequest.cs

```diff
@@ -7,7 +7,29 @@ namespace GitHub.Actions.RunService.WebApi
     [DataContract]
     public class AcquireJobRequest
     {
+        [DataMember(Name = "jobMessageId", EmitDefaultValue = false)]
+        public string JobMessageID { get; set; }
+
+        // The earliest versions of this type serialized the JobMessageID under a field named "streamId"
+        // This private property is expected to be a short-lived shim to aid in transitioning over to the new schema.
         [DataMember(Name = "streamId", EmitDefaultValue = false)]
-        public string StreamID { get; set; }
+        private string DeprecatedJobMessageIdAlias
+        {
+            get
+            {
+                return this.JobMessageID;
```

Comment:
"streamId" gets serialized out with whatever value is stored in this.JobMessageID 

(Note that it doesn't have its own backing field.  It piggy-backs on JobMessageID.)